ada_2_fhir-r4/zibs2020/payload/0.6-dev/ext-TimeInterval.xsl

```diff
@@ -49,97 +49,43 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                     </xsl:when>
 
                     <xsl:otherwise>
-                        <!-- Convert input to xs datatypes -->
-                        <!-- this does not support T-dates -->
-                        <xsl:variable name="startDateTime" select="
-                                if (start_datum_tijd[@value]) then
-                                    nf:timestamp-to-dateTime(start_datum_tijd/@value)
-                                else
-                                    false()"/>
-                        <xsl:variable name="endDateTime" select="
-                                if (eind_datum_tijd[@value]) then
-                                    nf:timestamp-to-dateTime(eind_datum_tijd/@value)
-                                else
-                                    false()"/>
-                        <xsl:variable name="duration" select="
-                                if (tijds_duur[@value]) then
-                                    nf:quantity-to-xsDuration(tijds_duur)
-                                else
-                                    false()"/>
-
-                        <!-- Calculate the "picture" to format the date. Only handle two cases of precisions: dates (year,
-                             month, day) and dateTimes (with second precision. -->
-                        <xsl:variable name="picture">
-                            <xsl:choose>
-                                <xsl:when test="
-                                        start_datum_tijd/@value castable as xs:date or
-                                        concat(start_datum_tijd/@value, ':00') castable as xs:date or
-                                        eind_datum_tijd/@value castable as xs:date or
-                                        concat(eind_datum_tijd/@value, ':00') castable as xs:date">[Y0001]-[M01]-[D01]</xsl:when>
-                                <xsl:otherwise>[Y0001]-[M01]-[D01]T[H01]:[m01]:00[Z]</xsl:otherwise>
-                            </xsl:choose>
-                        </xsl:variable>
-
-                        <!-- Get start from input or calculate it from end and duration -->
-                        <xsl:variable name="start">
-                            <xsl:choose>
-                                <xsl:when test="start_datum_tijd[@value]">
-                                    <!-- commented out, does not support T-dates -->
-                                    <!--<xsl:value-of select="format-dateTime($startDateTime, $picture)"/>-->
+                        <!-- Make fhir format of the ada input -->
```

Comment:
Dat deze wijziging nog in 0.6-dev zit is wel een teken dat deze branch behoorlijk achterloopt. Dat stond bij mij op m'n lijstje om allemaal recht te trekken als de MP907-wijzigingen goedgekeurd zijn (want ik wel 0.5 en 0.6 naar master hebben gezien dat gepubliceerde packages zijn)

Reply:
Hmmm, ik heb niet naar master gekeken idd. Alleen naar de MP920 branche, maar dacht dat we daar master wel regelmatig in gesynched hadden eigenlijk.

Reply:
Niet iets om hier op te lossen